analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -344,13 +344,13 @@ private fun resolveConfiguration(
     val packageIds = analyzerResult.packages.mapTo(mutableSetOf()) { it.id }
     val packageCurations = mutableListOf<PackageCuration>()
 
-    val duration = measureTime {
-        curationProviders.forEach { (_, curationProvider) ->
+    curationProviders.forEach { (id, curationProvider) ->
+        val duration = measureTime {
             packageCurations += curationProvider.getCurationsFor(packageIds).values.flatten()
         }
-    }
 
-    Analyzer.logger().debug { "Getting package curations took $duration." }
+        Analyzer.logger().info { "Getting package curations from provider '$id' took $duration." }
```

Comment:
While at it, would also make sense to log the number of curations retrieved from each provider?